### PR TITLE
Update introduction-how-it-works.md

### PR DESCRIPTION
**Pull request by ifaqeer-zz**
Added line #24 for added clarity

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc_source/introduction-how-it-works.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Global Accelerator supports both TCP and UDP protocols\.
...
 + AWS Direct Connect does not advertise IP address prefixes for AWS Global Accelerator over a public virtual interface\. We recommend that you do not advertise IP addresses that you use to communicate with Global Accelerator over your AWS Direct Connect public virtual interface\. If you advertise IP addresses that you use to communicate with Global Accelerator over your AWS Direct Connect public virtual interface, it will result in an asymmetric traffic flow: your traffic toward Global Accelerator goes to Global Accelerator over the internet, but return traffic coming to your on\-premises network comes over your AWS Direct Connect public virtual interface\.
 + Global Accelerator does not support processing IP packet fragments or re\-assembly\. An intermediate router or gateway operating at layer 3 might fragment a packet into multiple smaller packets between the client and the Global Accelerator endpoint\. If that happens, the fragments are not processed or re\-assembled by Global Accelerator and are not delivered to the endpoint\.
++ Global Accelerator does not support resources/endpoints which reside in different AWS accounts.
```
Comment (staciasnapp) [inline doc_source/introduction-how-it-works.md:27]: Sounds good to add a line here about this, to make sure people are aware that they can't use resources in multiple AWS accounts. How about this? "Global Accelerator does not support adding as endpoints resources that belong to other AWS accounts."

Comment (ifaqeer-zz) [reply doc_source/introduction-how-it-works.md:27]: Make sense. Thank you

Comment (staciasnapp) [reply doc_source/introduction-how-it-works.md:27]: Do you want to update your PR, and then I'd be happy to merge it in?

**Comment by ifaqeer-zz** [comment]
Not sure how/where would I goto update the PR for this one?. Never had to do it before for other similar changes.

**Comment by staciasnapp** [comment]
Ah. Well, I can just go ahead and merge in your change, and then I'll update it next time I post a refreshed version to GitHub. Thanks!

**Pull request merged**